lib/csv/csv.nit

```diff
@@ -103,7 +103,7 @@ class CsvDocument
 	# * `has_header`: Is the first row the header?
 	# * `skip_empty`: Do we skip the empty lines?
 	# For details, see `CsvReader.skip_empty`.
-	fun load_from(stream: IStream, has_header: Bool, skip_empty: Bool) do
+	fun load_from(stream: Reader, has_header: Bool, skip_empty: Bool) do
```

Comment:
StreamReader?

`Reader` seems a pretty abstract word for this case. Furthermore, I think simple words like this one should stay available for Nit users in their libs.